.gitignore

```diff
@@ -12,3 +12,4 @@ vendor/
 .byebug_history
 /tmp
 *.iml
+auto_ocp4.sh
```

Comment:
What is this file? Do we need an ignore for it?

Reply:
we set env for testing in local using this , I needed to ignore it , but better option is to not track it by keeping it in other directory , may cause issue for others in future in case some file with this name is needed in the project , have reverted the gitignore file to original